src/client/graphql/QuoteCart.graphql

```diff
@@ -1,3 +1,137 @@
+fragment QuoteDataFragment on BundledQuote {
```

Comment:
Yeah I'm not sure how to make that work - like imports between files -- do you know?

Reply:
I think if you put the fragment in another `.graphql` file it seems like it will automatically be available in all other `.graphql` files. So we could create a `/fragments` folder in `/graphql` and put all the fragments there. Maybe with some naming convention of suffixing the file names with `.fragment`? Like `QuoteData.fragment.graphql`.

Reply:
Smart! Will fix that :D

Reply:
Awesome! 🙌 ✨